Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM golang:1.14-alpine as builder
```

Comment:
1.14 is the latest for alpine?

Reply:
Bumping to latest version for the builder